app/util/ExpiryPoller.scala

```diff
@@ -0,0 +1,56 @@
+package util
+
+import java.util.Date
+import javax.inject.Inject
+import akka.actor.Scheduler
+import data.AuditDataStore
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import com.gu.atom.publish.{LiveAtomPublisher, PreviewAtomPublisher}
+import model.commands.{PublishAtomCommand, UpdateAtomCommand}
+import com.gu.atom.data.{PublishedDataStore, PreviewDataStore}
+import com.gu.pandomainauth.model.{User => PandaUser}
+
+case class ExpiryPoller @Inject () (
+                        implicit val previewDataStore: PreviewDataStore,
+                        implicit val publishedDataStore: PublishedDataStore,
+                        previewPublisher: PreviewAtomPublisher,
+                        livePublisher: LiveAtomPublisher,
+                        youtubeConfig: YouTubeConfig,
+                        implicit val auditDataStore: AuditDataStore,
+                        val awsConfig: AWSConfig
+                       ){
+  def start(scheduler: Scheduler): Unit = {
+
+    scheduler.schedule(0.seconds, 6.hours)(checkExpiryDates())
+
+  }
+
+  def checkExpiryDates(): Unit = {
+
+    val timeNow = new Date().getTime
+    implicit val username = PandaUser(awsConfig.expiryPollerName,
+      awsConfig.expiryPollerLastName, "expiryPoller", None)
+
+    previewDataStore.listAtoms.map(atoms => {
```

Comment:
I'd rather do these checks in the updateStatusIfExpired function so that I can use the same function for checking atoms when atom metadata gets changed.